src/internal/config.js

```diff
@@ -151,6 +151,7 @@ module.exports = {
     manageInvoiceAccounts: true,
     deleteAllBillingData: ['local', 'dev', 'development', 'test', 'qa'].includes(process.env.NODE_ENV),
     waterAbstractionAlerts: true,
-    recalculateBills: true
+    recalculateBills: true,
+    allowChargeVersionUploads: process.env.ALLOW_CHARGE_VERSION_UPLOADS || false
```

Comment:
Check out `!!`, e.g. `allowChargeVersionUploads: !!process.env.ALLOW_CHARGE_VERSION_UPLOADS`

Reply:
Good catch, not sure why I did that.

Reply:
Changed as requested